
scala> def f = throw new Exception("Uh-oh")
f: Nothing

scala> f
java.lang.Exception: Uh-oh
  at .f(<console>:1)
  ... ??? elided

scala> def f = throw new Exception("")
f: Nothing

scala> f
java.lang.Exception:
  at .f(<console>:1)
  ... ??? elided

scala> def f = throw new Exception
f: Nothing

scala> f
java.lang.Exception
  at .f(<console>:1)
  ... ??? elided

scala> null.asInstanceOf
java.lang.NullPointerException
  at .$print$lzycompute(<synthetic>:8)
  ... ??? elided

scala> :quit
